tracer/src/Datadog.Tracer.Native/cor_profiler.cpp

```diff
@@ -949,7 +971,8 @@ HRESULT CorProfiler::TryRejitModule(ModuleID module_id)
                     // Check if the typeref matches
                     mdToken parent_token = mdTokenNil;
                     mdToken attribute_ctor_token = mdTokenNil;
-                    const void* attribute_data = nullptr; // Pointer to receive attribute data, which is not needed for our purposes
+                    const void* attribute_data = nullptr;
+                    // Pointer to receive attribute data, which is not needed for our purposes
```

Comment:
The reformatting makes this comment confusing. Maybe you could move this up a line?

Reply:
yes! fixed in b9b2c8c95e0dc5c966184933b404f5d1ed7d06ba